glotaran/builtin/io/pandas/xlsx.py

```diff
@@ -0,0 +1,49 @@
+"""Register XLSX parameters."""
+
+from __future__ import annotations
+
+import numpy as np
+import pandas as pd
+
+from glotaran.io import ProjectIoInterface
+from glotaran.io import register_project_io
+from glotaran.parameter import ParameterGroup
+from glotaran.utils.io import safe_parameters_fillna
+from glotaran.utils.io import safe_parameters_replace
+
+
+@register_project_io(["xlsx", "ods"])
+class ExcelProjectIo(ProjectIoInterface):
```

Comment:
Technically, specifically the xlsx (the Office Open XML standard) and ods (the Open Document Standard) format are not specific to Excel, but open standards. So the class could be named differently, but since it uses the read_excel function from pandas under the hood, who are we to change the name?  
Note that the pandas `read_excel` function supports xls, xlsx, xlsm, xlsb, odf, ods and odt, so more than just xlsx and ods.